morse_code.rb

```diff
@@ -0,0 +1,25 @@
+def get_char(char)
+  morse_dict = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+    '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S',
+    '-' => 'T', '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse_dict[char]
+end
+puts get_char('.-') #=> A
+def get_word(arr)
+  word = ''
+  arr.split.each { |i| word += get_char(i) }
+  word
+end
+puts get_word('-- -.--') #=> MY
+
+def decode(str)
+  sentese = ''
+  arry = str.split('   ')
+  arry.each { |i| sentese += "#{get_word(i)} " }
+  sentese
+end
+x = decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
+puts x
```

Comment:
- [ ] As stated in the project requirement Kindly I recommend you Create a method to decode the entire message in Morse code,   `decode("-- -.--   -. .- -- .")` that return **"MY NAME"**, when i checked out your code the result I get is the below but it must be like this :
```
"A"
"MY"
"MY NAME"
"A BOX FULL OF RUBIES"
```


![Capture](https://user-images.githubusercontent.com/93598189/216494979-4d55c849-8958-4d2b-ab6c-1e4396485ecc.PNG)